src/ssh/azext_ssh/resource_tag_utils.py

```diff
@@ -0,0 +1,107 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core import azclierror
+from knack import log
+
+logger = log.get_logger(__name__)
+# Checking Port and Resource Tag argument and fetching appropiate tag
+
+
+def _handle_os_type_for_tag(cmd, op_info):
```

Comment:
The name of this function doesn't describe what the function is, we want to keep the name as descriptive as possible. Something like "_get_port_from_az_resource_tag" would make more sense.

Reply:
Yes, that makes sense. Some sample names could be:
-Configure_port_From_Resource_Tag
-Set_port_From_Resource_Tag

Reply:
I like configure_port_from_resource_tag